docs/plugins/language.md

```diff
@@ -147,42 +143,42 @@ Language.set(languageCode).then(() => {
 }).catch()
 ```
 
-The `set`-method returns a Promise, that resolves when the Language has been properly changed.
-Setting the language is _instant_ for translations specified inline in the `translations.json` file.
-In the case of external language files an _asynchronous_ load request will be made.
+The `set` method returns a promise that resolves when the `Language` has been properly changed.
+Setting the language is *instant* for translations specified inline in the **translations.json** file.
+For external language files, an *asynchronous* load request will be made.
 
-The language code should match with one of the language-keys specified in your `translations.json`.
+The language code should match with one of the language keys specified in your **translations.json**.
 
-The `map` option in the `meta` key of your translations file can be used to specify _aliases_, and as such can be used
-to map different ISO codes (i.e. en-GB, en-US, etc.) to a single language key (en).
+The `map` option in the `meta` key of your translations file can be used to specify *aliases*, and as such can be used
+to map different ISO codes (i.e., en-GB, en-US) to a single language key (en).
```

Comment:
Yep. See [https://www.instructionalsolutions.com/blog/how-to-use-i-e-and-e-g](https://www.instructionalsolutions.com/blog/how-to-use-i-e-and-e-g). 

By the way, the correct acronym here should be 'e.g.' instead of 'i.e.', because the languages are merely examples. Changed accordingly.

Reply:
cool .. every day is a school day :)